13-TeV-examples/uproot_python/HyyAnalysisNew.ipynb

```diff
@@ -164,526 +164,551 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "[Back to contents](#contents)"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "<a id='fraction'></a>"
+    "## Example 1: Reading data "
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Lumi, fraction, file path\n",
-    "\n",
-    "General definitions of luminosity, fraction of data used, where to access the input files"
+    "We would like to read some of the data from the open dataset."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 15,
    "metadata": {},
    "outputs": [],
    "source": [
-    "lumi = 36.1 # fb-1 # data\n",
+    "# CERN open database URL\n",
```

Comment:
Probably "CERN Open Data Portal URL"? (Database --> Data is the main thing I'd suggest to change there)

Reply:
Indeed it's not, it's in the ATLAS Open Data directory (even though in the future we should have the ntuples also/only in the CERN Open Data portal

Reply:
Then I'll update it to "ATLAS Open Data directory"